src/org/javarosa/core/io/StreamsUtil.java

```diff
@@ -5,22 +5,20 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
+/**
+ * @deprecated Please, don't use anything from this class **at all**. It will removed on the next release
```

Comment:
I think the deprecation annotation makes the intent clear but if you want to add a comment, perhaps "This class is unused and should remain that way. It will be removed in a future release.

Reply:
OK, I'll add the comments on deprecated classes.

This will be my last allegation in favor of breaking the build if there are compilation warnings :) 

Remember that we're already breaking the build for code styling reasons... breaking it for using deprecated stuff makes all the sense to me. Also, what risks do you see, @lognaturel? Maybe I'm missing something...

Right now, there's no warnings at all (I've solved them all). It would be really easy to roll this into the build :)